specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/LongTermRetentionBackups.json

```diff
@@ -1015,6 +1156,24 @@
     }
   },
   "definitions": {
+    "ChangeLongTermRetentionBackupAccessTierParameters": {
+      "description": "Contains the information necessary to change long term retention backup access tier and realted operation mode.",
+      "required": [
+        "backupStorageAccessTier",
+        "operationMode"
+      ],
+      "type": "object",
+      "properties": {
+        "backupStorageAccessTier": {
+          "description": "The long term retention backup storage access tier",
+          "type": "string"
+        },
+        "operationMode": {
+          "description": "The operation mode when updating ltr backup storage access tier",
+          "type": "string"
+        }
```

Comment:
Should these be enums?

Reply:
This should be string here because it is a different property.